router/routes/courses.js

```diff
@@ -589,6 +589,7 @@ function  generateListings(data, user, cb) {
         if (debug || (user != '' && user != undefined)) {
             var classid = "ClassTranscribe::Course::" + e['id'];
             acl.isAllowed(user, classid, 'Modify', function (err, res) {
+                res = true
```

Comment:
Why is being res set to true?

Reply:
res is set true for testing purpose. So we may have the permission of managing courses.

Reply:
I have removed "res = true".